packages/ecommerce-app-base/src/SkuPicker/ProductList/ProductList.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import tokens from '@contentful/f36-tokens';
+import { css } from 'emotion';
+import { Product } from '../../types';
+import { ProductListItem } from './ProductListItem';
+
+export interface Props {
+  products: Product[];
+  selectProduct: (sku: string) => void;
+  selectedSKUs: string[];
+}
+
+const styles = {
+  productList: css({
+    display: 'flex',
+    flexWrap: 'wrap',
+    marginLeft: `-${tokens.spacingS}`,
+    marginRight: `-${tokens.spacingS}`,
+  }),
+};
+
+export function ProductList({ selectProduct, selectedSKUs, products }: Props) {
```

Comment:
one visual comment on this one: the images look very stretched vertically, is this intentional? 
![Screenshot 2023-08-24 at 3 49 45 PM](https://github.com/contentful/apps/assets/58186851/25d7ffd9-19f8-4377-801b-cc1896945ae5)

Reply:
@lilbitner I didn't touch the implementation, just added the stories. It might be that the layout container causes this effect - and it doesn't happen when loaded in the web app.

Reply:
ah okay, that makes sense 👍